cmd/execution_builder.go

```diff
@@ -735,19 +738,12 @@ func (exeNode *ExecutionNode) LoadIngestionEngine(
 	return exeNode.ingestionEng, err
 }
 
-func (exeNode *ExecutionNode) LoadFollowerEngine(
+func (exeNode *ExecutionNode) LoadConsensusCommittee(
```

Comment:
Previously we didn't start consensus committee it kinda worked in integration tests but generally speaking we would be missing some events without it